tools/hicup/hicup_deduplicator.xml

```diff
@@ -0,0 +1,43 @@
+<tool id="hicup_deduplicator" name="Deduplicator" version="0.1.0">
+    <requirements>    
+        <requirement type="package" version="0.5.9">hicup</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command><![CDATA[
+        hicup_deduplicator $input_file
+        --threads $advanced_options.threads
+        $advanced_options.zip
+    ]]></command>
+    <inputs>
+        <param name="input_file" type="data" format="bam,sam" label="Input file(s)" help="Input files in bam or sam format."/>
+        <section name="advanced_options" title="Advanced options">
+            <param argument="--threads" type="integer" value="1" label="Threads" help="Specify the number of threads, allowing simultaneous processing of multiple files"/>
+            <param argument="--zip" type="boolean" value="false" truevalue="--zip" falsevalue="" label="Compress output" help="Compress the output with zip"/>
+        </section>
+    </inputs>
+    <outputs>
+        <data format="txt" name="digest_file">
+            <discover_datasets pattern="__designation_and_ext__" visible="true" />
+        </data>
+    </outputs>
+    <help><![CDATA[
```

Comment:
Test are an issue. But e.g. I need a bowtie index and the complete genome. If I am using the provided test data I need about 3 GB of space.